app/views/users/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<span id="profile">
+  <h1><%= @user.username %></h1>
+  <h2>About Me</h2>
+  <p id="bio"><%= @user.bio %></p>
+  <h2>My Ideas</h2>
+  <ul class="idea_list">
+  <% @user.ideas.each do |idea| %>
+    <li><%= link_to(idea.title, idea_path(idea)) %>
+    <div id="comment-info"> <%= idea.comments.count %> <i class="fa fa-comments"></i></div></li>
```

Comment:
It's style nitpicking, but I'm typically opposed to packing as much content into one line as you are in this file.

```
<p id="bio">
  <%= @user.bio %>
</p>
```
is more readable than 

`<p id="bio"><%= @user.bio %></p>`

And ..

```
<li>
  <%= link_to(idea.title, idea_path(idea)) %>
  <div id="comment-info"> 
    <%= idea.comments.count %> <i class="fa fa-comments"></i>
  </div>
</li>
```
Makes it easier to tell what is going on than..

```
<li><%= link_to(idea.title, idea_path(idea)) %>
<div id="comment-info"> <%= idea.comments.count %> <i class="fa fa-comments"></i></div></li>
```